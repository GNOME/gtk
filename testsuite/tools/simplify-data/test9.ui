<!-- non-default child property remains -->
<interface>
  <object class="GtkGrid">
    <child>
      <object class="GtkLabel">
        <layout>
          <property name="left-attach">0</property>
          <property name="top-attach">0</property>
        </layout>
      </object>
    </child>
  </object>
</interface>
