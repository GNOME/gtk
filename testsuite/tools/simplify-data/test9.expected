<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkGrid">
    <child>
      <object class="GtkLabel">
        <layout>
          <property name="left-attach">0</property>
          <property name="top-attach">0</property>
        </layout>
      </object>
    </child>
  </object>
</interface>
